### PR TITLE
new case development:boot guest after disable ept/npt

### PR DESCRIPTION
**Pull request by leidwang**
boot_with_disable_ept:boot guest after disable ept/npt

Signed-off-by: Leidong Wang <leidwang@redhat.com>

ID:1905332

### CODE REVIEW

**Comment by leidwang** [comment]
JOB ID     : 6cd7d36f9d77d9ab4a30b1ece1fe33dbe79abbd7
JOB LOG    : /root/avocado/job-results/job-2020-12-08T00.21-6cd7d36/job.log
 (1/1) type_specific.io-github-autotest-qemu.boot_with_disable_ept: PASS (95.33 s)

**Comment by leidwang** [comment]
@XueqiangWei Could you please review this patch?Thanks!

**Comment by XueqiangWei** [comment]
@leidwang both ovmf and seabios all support it, so please update the title \"ovmf new case development\"

**Review by XueqiangWei — COMMENTED** [review]
Since windows guest is blocked by product bug, I just test linux guest.
Tested on amd and intel host, not hit any issue.
(1/1) type_specific.io-github-autotest-qemu.boot_with_disable_ept: PASS (83.88 s)

LGTM,
Acked-by: xuwei@redhat.com

**Comment by leidwang** [comment]
@vivianQizhu Could you please review this patch? Thanks!

File: qemu/tests/cfg/boot_with_disable_ept.cfg [hunk 1]
```diff
@@ -0,0 +1,17 @@
+- boot_with_disable_ept:
+    virt_test_type = qemu
+    type = boot_with_disable_ept
+    start_vm = no
+    HostCpuVendor.intel:
+        parameter = 'ept'
```
Comment (vivianQizhu) [inline qemu/tests/cfg/boot_with_disable_ept.cfg:6]: Please cautiously use such keyword, recommend to name it `parameter_name`.

File: qemu/tests/boot_with_disable_ept.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+
+    def check_parameter_status():
+        output = process.getoutput(params["check_status_cmd"])
+        if output != params["expected_status"]:
+            test.fail("Disable %s failed" % params["parameter"])
+
+    check_parameter_status()
```
Comment (vivianQizhu) [inline qemu/tests/boot_with_disable_ept.py:23]: I suppose here we don't need to define this function, the total `run()` function is very short and we don't use this function any other where.

File: qemu/tests/boot_with_disable_ept.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    params["start_vm"] = 'yes'
+    env_process.preprocess_vm(test, params, env, params["main_vm"])
+    vm = env.get_vm(params['main_vm'])
+    timeout = int(params.get("login_timeout", 360))
```
Comment (vivianQizhu) [inline qemu/tests/boot_with_disable_ept.py:27]: Would you help illustrate the specific reason that you tolerant such a long timeout for the login action?

Comment (leidwang) [reply qemu/tests/boot_with_disable_ept.py]: There is no special reason, will use the default timeout value, the code has been updated. thank you very much.

**Review by vivianQizhu — APPROVED** [review]

**Pull request merged**